airbyte_cdk/test/standard_tests/docker_base.py

```diff
@@ -145,23 +155,16 @@ def test_docker_image_build_and_spec(
                 no_verify=False,
             )
 
-        try:
-            result: CompletedProcess[str] = run_docker_command(
-                [
-                    "docker",
-                    "run",
-                    "--rm",
-                    connector_image,
-                    "spec",
-                ],
-                check=True,  # Raise an error if the command fails
-                capture_stderr=True,
-                capture_stdout=True,
-            )
-        except SubprocessError as ex:
-            raise AssertionError(
-                f"Failed to run `spec` command in docker image {connector_image!r}. Error: {ex!s}"
-            ) from None
+        _ = run_docker_airbyte_command(
```

Comment:
Basically everywhere that we were calling `run_docker_command` for running an Airbyte command (meaning, except actions like "docker build" and "docker tag"), we now use `run_docker_airbyte_command`, and we get better error messaging with a lot less work.